python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1122,19 +1122,19 @@ def convert_dropout(node, **kwargs):
     """
     from onnx.helper import make_node
     name, input_nodes, attrs = get_inputs(node, kwargs)
-    opset_version = kwargs["opset_version"]
 
-    probability = float(attrs.get("p", 0.5))
+    _ = float(attrs.get("p", 0.5))
+    _ = convert_string_to_list(attrs.get("axes", "None"))
+    mode = attrs.get('mode', 'training')
 
-    if opset_version >= 12:
-        # opset >= 12 requires the ratio to be an input
-        nodes = [
-            create_const_scalar_node(name+"_ratio0", np.float32(probability), kwargs),
-            make_node("Dropout", [input_nodes[0], name+"_ratio0"], [name], name=name)
-        ]
-        return nodes
-    else:
-        return [make_node("Dropout", input_nodes, [name], ratio=probability, name=name)]
+    if mode != 'training':
```

Comment:
Do we want to check and preserve the Dropout nodes if mode is training? Otherwise if inference mode, return identity op.

Reply:
I think dropout is almost never used in inference because the output values will be randomly dropped and theoretically it's designed for training only. I think not supporting 'both' mode, at least for now, can confirm this while we export the models in the model zoo.

Reply:
Sorry, just trying to understand the mode parameter and how it applies to these training-only ops. Is mode set to 'training' in the models or are you just depending on the default from attrs.get() above? 

Just wondering for future plans to export models for training (if needed.) Do we need to specify whether we are exporting for training or for inference in the export call, or can we make it operator dependent?

Reply:
Feel free to merge, we should just continue this conversation for future uses.

Reply:
> Sorry, just trying to understand the mode parameter and how it applies to these training-only ops. Is mode set to 'training' in the models or are you just depending on the default from attrs.get() above?
> 
> Just wondering for future plans to export models for training (if needed.) Do we need to specify whether we are exporting for training or for inference in the export call, or can we make it operator dependent?

`mode` is a parameter to MXNet operator `Dropout` only; it's now a parameter for the model :)